pandas-stubs/core/indexes/timedeltas.pyi

```diff
@@ -34,9 +34,9 @@ class TimedeltaIndex(DatetimeTimedeltaMixin, TimedeltaIndexProperties):
     def __truediv__(self, other: num) -> TimedeltaIndex: ...  # type: ignore[override]
     def astype(self, dtype, copy: bool = ...): ...
     def get_value(self, series, key): ...
-    def get_loc(self, key, tolerance=...): ...
+    def get_loc(self, key, method=..., tolerance=...): ...
```

Comment:
Why are you adding this back in? This was effectively deprecated in 1.4

Reply:
Thank you! I went through the error messages of stubtest - which unfortunately cannot know what is deprecated.